onnxruntime/core/graph/function.cc

```diff
@@ -242,11 +303,9 @@ FunctionImpl::FunctionImpl(const onnxruntime::Graph& graph,
       auto iter = input_name_idx_map.find(tensor_name);
       if (iter != input_name_idx_map.end()) {
         // Preserving NodeArg and input/output names
-        ONNX_NAMESPACE::NodeProto temp_node_proto;
-        node_in_parent_graph->ToProto(temp_node_proto);
-        const onnxruntime::NodeArg* node_arg = parent_graph_->GetNodeArg(temp_node_proto.input().Get(input_name_idx_map[tensor_name]));
+        const onnxruntime::NodeArg* node_arg = parent_graph_->GetNodeArg(function_op_node_proto.input().Get(input_name_idx_map[tensor_name]));
```

Comment:
>input_name_idx_map[tensor_name] [](start = 108, length = 31)

Could use *iter instead of doing another lookup.

Reply:
Hmm I fixed the same issue in a couple of other places - will fix this too

Reply:
Fixed.